### PR TITLE
[GSB,mkt] log level decrease

### PR DESCRIPTION
**Pull request by tworec**

### CODE REVIEW

File: service-bus/router/src/lib.rs [hunk 1]
```diff
@@ -182,16 +182,20 @@ where
             }
         } else {
             match self.registered_endpoints.entry(msg.service_id.clone()) {
-                Entry::Occupied(_) => RegisterReply {
-                    code: RegisterReplyCode::RegisterConflict as i32,
-                    message: format!("Service ID '{}' already registered", msg.service_id),
-                },
+                Entry::Occupied(_) => {
+                    log::warn!("Service '{}' already registered", msg.service_id);
```
Comment (mfranciszkiewicz) [inline service-bus/router/src/lib.rs:186]: Won't this effectively print the message twice (for local router)?

File: core/market/src/protocol/negotiation/provider.rs [hunk 2]
```diff
@@ -170,7 +170,7 @@ impl NegotiationApi {
         caller: String,
         msg: ProposalReceived,
     ) -> Result<(), CounterProposalError> {
-        log::debug!(
+        log::trace!(
```
Comment (nieznanysprawiciel) [inline core/market/src/protocol/negotiation/provider.rs:173]: Do we log anywhere, that we received Proposal from other party?

File: service-bus/router/src/lib.rs (hunk 1 above)
Comment (tworec) [reply service-bus/router/src/lib.rs:186]: Indeed. Reverting this.

File: core/market/src/protocol/negotiation/provider.rs (hunk 2 above)
Comment (tworec) [reply core/market/src/protocol/negotiation/provider.rs:173]: We had this information tripled
```
[2020-11-05T21:41:06Z DEBUG ya_net::service] Incoming msg from = 0xe0489275bd2f5fce6ead8256382615213087bef7, to = /net/0xb11cd0514d0b33251345b12387cb893a0d75f1f5/market/protocol/mk1/negotiation/requestor/proposal/ProposalReceived, fwd to local addr = /public/market/protocol/mk1/negotiation/requestor/proposal/ProposalReceived, request_id: 6340869910067749
[2020-11-05T21:41:06Z DEBUG ya_market::protocol::negotiation::requestor] Negotiation API: Received proposal [P-a6e9fdd7669bbeb383af0b2326bd1dd20fd4a1ea28d411f7ea78f5827186374b] from [0xe0489275bd2f5fce6ead8256382615213087bef7].
[2020-11-05T21:41:06Z INFO  ya_market::negotiation::common] Received counter Proposal [R-a6e9fdd7669bbeb383af0b2326bd1dd20fd4a1ea28d411f7ea78f5827186374b] for Proposal [R-8ace9eaf3a236b36c97249c8714076c575a4c9478824e7a73d738f3cffed1e3c] from [0xe0489275bd2f5fce6ead8256382615213087bef7].
```

Thats why I've changed DEBUG to TRACE for first two of them

**Review by nieznanysprawiciel — APPROVED** [review]

**Review by mfranciszkiewicz — APPROVED** [review]

**Review by nieznanysprawiciel — APPROVED** [review]

**Pull request merged**